src/lib_ccaligner/params.cpp

```diff
@@ -495,31 +499,14 @@ void Params::inputParams(int argc, char *argv[])
 void Params::validateParams()
 {
     if(audioFileName.empty() && !readStream)
+    {
         FATAL(EXIT_INVALID_PARAMETERS, "Audio file name is empty!");
+    }
 
     if(subtitleFileName.empty())
+    {
         FATAL(EXIT_INVALID_PARAMETERS, "Subtitle file name is empty!");
-
-    if(modelPath.empty())
```

Comment:
Why are these logs deleted?

Reply:
@saurabhshri Because they are never called. Those paths are always initialized in the constructor, and can never be set to empty. Should I still keep them?